### DISCUSSION TITLE
Is _DRIVER_TEAM_MAPPING in init.py still needed?

### DISCUSSION THREAD

**Discussion by Casper-Guo** [theOehrly/Fast-F1#587] [General]
It is a dictionary mapping driver numbers to some basic driver information, e.g.

```
'23': {'Abbreviation': 'ALB', 'FirstName': 'Alexander',
           'LastName': 'Albon', 'TeamName': 'Williams'},
```

It is used in `core.py` for when driver data is unavailable when loading laps.

The mapping is outdated. If we still rely on it then a robust solution should be pursued, if we no longer use it then it can be removed.

**Comment by theOehrly** [comment]
I'd say remove it. The variable is technically is still accessed. But the whole code block that uses it isn't reached usually. It only runs if the F1 API isn't supported or has no driver info, **and** if Ergast has no result data as well. I think this was introduced when the livetiming API wasn't working at all and loading data from recordings was the only option.
If this were still used, we'd have seen issues created due to incorrect driver/team info over the last year at least. Therefore, I don't think anything speaks against just removing it?
